src/main/java/com/github/princesslana/smalld/SmallD.java

```diff
@@ -63,6 +66,19 @@ public String getToken() {
     return token;
   }
 
+  public int getCurrentShard() {
+    return currentShard;
+  }
+
+  public int getNumberOfShards() {
+    return numberOfShards;
+  }
+
+  public void setShard(int current, int number) {
```

Comment:
Should we put setters for the current shard and the shard number? It's not a big deal, but just for exposing sake and letting the user what they want to set and when. This method is good, just think we could setup setters for the others as well.

Reply:
If a use case for just setting one could be made, I'd add them, but otherwise I don't see where you would be setting one without also knowing the other.

Reply:
Making mock copies of the smalld data, then setting the shard number to abstract from it would be a case I can think of, only setting the second would be a case for the first shard, so if default is 0 I'd only want to set the shard count.

Reply:
If it's the first shard, you know that current is zero.

I'm not sure what you're doing where you're making mock copies of the data and abstracting shard number?

Reply:
At the moment I'd be for leaving them out in the interest of having less methods.

Since I don't know how an auto-sharder would plug in to smalld, I'm open to adding them later if they were required/useful for that.